exporter/awsxrayexporter/internal/translator/http.go

```diff
@@ -33,62 +47,63 @@ func makeHTTP(span ptrace.Span) (map[string]pcommon.Value, *awsxray.HTTPData) {
 
 	span.Attributes().Range(func(key string, value pcommon.Value) bool {
 		switch key {
-		case conventions.AttributeHTTPMethod:
+		case conventions.AttributeHTTPMethod, AttributeHTTPRequestMethod:
 			info.Request.Method = awsxray.String(value.Str())
 			hasHTTP = true
-		case conventions.AttributeHTTPClientIP:
+		case conventions.AttributeHTTPClientIP, AttributeClientAddress:
+			//urlParts[conventions.AttributeHTTPClientIP] = value.Str()
 			info.Request.ClientIP = awsxray.String(value.Str())
 			info.Request.XForwardedFor = aws.Bool(true)
 			hasHTTP = true
-		case conventions.AttributeHTTPUserAgent:
+		case conventions.AttributeHTTPUserAgent, AttributeUserAgentOriginal:
 			info.Request.UserAgent = awsxray.String(value.Str())
 			hasHTTP = true
-		case conventions.AttributeHTTPStatusCode:
+		case conventions.AttributeHTTPStatusCode, AttributeHTTPResponseStatusCode:
 			info.Response.Status = aws.Int64(value.Int())
 			hasHTTP = true
-		case conventions.AttributeHTTPURL:
-			urlParts[key] = value.Str()
+		case conventions.AttributeHTTPURL, AttributeURLFull:
+			urlParts[conventions.AttributeHTTPURL] = value.Str()
```

Comment:
are you going to update the new convention keys (in urlParts) later in a separate PR?

Reply:
I don't know if that's necessary, it looks like the construct logic relies on the non-stable keys. The PR continues to use the non stable attributes and keys even if the stable attribute was present in the span. It would probably make sense to switch over to primarily using the stable semantic conventions when when the new `semconv` packages are released.

Reply:
make sense!

Reply:
Existing function `constructClientURL` and `constructServerURL` still use old conventions, we will update them to use new conventions when the stable semconv are involved in Collector.